src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -6,8 +6,6 @@
    [utils.re-frame :as rf]
    [quo2.foundations.colors :as colors]
    [status-im2.contexts.communities.overview.style :as style]
```

Comment:
please keep ;; TODO move to status-im2 when reimplemented, this comment for all requirement under it

Reply:
I decided to remove the code that needed these imports as it was just for mock data. So no more status_im in this file 🧹